src/usr/lib/ztp/dhcp/inband-ztp-ip

```diff
@@ -18,6 +18,8 @@
 # Purpose:      Used by dhclient-script to configure IP address for front panel interface
 #
 
+export PATH=/usr/local/bin:$PATH
```

Comment:
Can you please explain why this change is needed?

Reply:
When this DHCP hook is being called, it executes CLI commands. CLI executable is under /usr/local/bin and it must be added to PATH in order for the hook to have access to the CLI executable. I consulted code owner from Broadcom about this issue and this solution was suggested.

Reply:
Understood.